inc/saiport.h

```diff
@@ -478,6 +486,15 @@ typedef enum _sai_port_attr_t
     SAI_PORT_ATTR_ADVERTISED_FEC_MODE,
 
     /**
+     * @brief Advertised port FEC AN request bit.  True means FEC is requested for 10/40/100G
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_ADVERTISED_FEC_MODE_REQUEST,
```

Comment:
what is the relationship between this one and SAI_PORT_ATTR_ADVERTISED_FEC_MODE? Is it true and the SAI_PORT_ATTR_ADVERTISED_FEC_MODE is valid only when SAI_PORT_ATTR_ADVERTISED_FEC_MODE_REQUEST is true?

should we add @validonly SAI_PORT_ATTR_ADVERTISED_FEC_MODE_REQUEST=true for attribute SAI_PORT_ATTR_ADVERTISED_FEC_MODE?